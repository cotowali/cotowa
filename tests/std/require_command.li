require_command('ls')
require_command('missing command')
