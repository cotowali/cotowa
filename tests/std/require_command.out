missing command: command not found
